openedx/core/djangoapps/content/learning_sequences/api/outlines.py

```diff
@@ -255,7 +256,7 @@ def get_content_errors(course_key: CourseKey) -> List[ContentErrorData]:
 
 @function_trace('learning_sequences.api.get_user_course_outline')
 def get_user_course_outline(course_key: CourseKey,
-                            user: User,
+                            user: Model,
```

Comment:
Whoops, @regisb I realize now you're talking specifically about the limitations of `get_user_model`. Yeah, you're right, that's a special weird case.

@ormsbee For such an edx-platform specific app like learning_sequences, would we lose much by just directly using `django.contrib.auth.models.User` instead of `get_user_model`?

Reply:
Can I suggest that we create a `types.py` somewhere in a common module? This types.py module would include things like:

```
import django.contrib.auth.models.User

User = django.contrib.auth.models.User
```

Then developers would write:

```
from openedx.core import types
def my_func(user: types.User): ...
```

Reply:
Can we hardcode it to `from django.contrib.auth import get_user_model` for this module only for now? I've honestly been contemplating removing the User object from this anyhow (and replacing it with another attrs class with fewer attributes).

Reply:
@ormsbee 
> Can we hardcode it to from django.contrib.auth import get_user_model for this module only for now?

Can you elaborate? Using `get_user_model` in learning_sequences would require us to annotate the user objects as type `Model` -- do you want to do that?